opentelemetry-semantic-conventions/scripts/templates/registry/rust/attribute.rs.j2

```diff
@@ -0,0 +1,24 @@
+{%- import 'macros.j2' as attr_macros -%}
+// DO NOT EDIT, this is an auto-generated file
+//
+// If you want to update the file:
+// - Edit the template at scripts/templates/registry/rust/attributes.rs.j2
+// - Run the script at scripts/generate-consts-from-spec.sh
+
+//! # Semantic Attributes
+//!
+//! The entire set of semantic attributes (or [conventions](https://opentelemetry.io/docs/concepts/semantic-conventions/)) defined by the project. The resource, metric, and trace modules reference these attributes.
+
+{% for root_ns in ctx %}
+    {% for attr in root_ns.attributes | rejectattr("name", "in", params.excluded_attributes) %}
+{{ [attr.brief, concat_if("\n\n# Notes\n\n", attr.note), attr_macros.examples(attr)] | comment }}
```

Comment:
Nit: Title/Heading1/`#` results in a kinda large tooltip, any thoughts about reducing the size of the headers here and elsewhere?

Reply:
Fixed in [4970ee9](https://github.com/open-telemetry/opentelemetry-rust/pull/2098/commits/4970ee9314ca5cb72376b21bb807ec14da0d4cc8)